### DISCUSSION TITLE
Using TEMPLATE STRING (w/ quasar TAG) as vue 3 "dynamic" template

### DISCUSSION THREAD

**Discussion by noxOfficer** [quasarframework/quasar#15414] [General - Components / Directives / etc] [answered]
Has anyone runs quasar tags recognized within a dynamic VUE template? 
It's running Ok with VUE regular tags. But not with QUASAR.

// just small exemple 
```
export default defineComponent({
    template: { `<h2>Hi {{name}}</h2>
                 <Q-BTN @click='test'>{{age}}</Q-BTN>
    ` },

    data() {
        return {
            name: 'Hommer',
            age: 24,
        }
    },
    methods: {
        test() {
            console.log('hello console')
        }
    }
})
```

// It must chance this flag "compiler" to true - and add 10Kb to final result hahaha - 
// this little one will add the compiler to properly do work - by default its false
// Using VUE -> vue.config.js module.exports = { runtimeCompiler: true };
// Using VUE+QUASAR -> quasar.config.js add at -> build: { vueCompiler: true, }

**Comment by metalsadman** [comment, marked as answer]
Auto import won't catch that, you need to register the said component in your quasar.config.js

**Reply by noxOfficer** [reply]
Thanks for helping us @metalsadman you spended your time to help community
Your info is a touch light, even because I have (quasar) components like q-upload and q-video that are non responding too. At development they work fine and on SPA notting even render - I guess maybe is the same behaviour, something what I need config?!
I will search something to register this components in quasar.config.js where I don't know. Do you know where?
Thanks again Metal Sandman
Plainfull gratiness.
